### DISCUSSION TITLE
HW4 Part 1-2

### DISCUSSION THREAD

**Discussion by cgallagher1021** [cooperrc/computational-mechanics#173] [Q&A]
Can anyone tell me why I am getting this error? 
<img width="653" alt="Hw4P1-2" src="https://user-images.githubusercontent.com/121522937/211956025-bc910110-aaa0-4f9f-bf32-9f835dea744b.PNG">

**Comment by Nadinemm5** [comment]
try printing D and Xn and see if they looks right. Look at this example from the reading:
![image](https://user-images.githubusercontent.com/121512414/211976837-599758f7-9d67-42ee-a42f-d076d0eed61d.png)
try `D = np.array([100, 0, 0, 0])`
